src/cli/cli_syntax.md

```diff
@@ -1,22 +1,143 @@
-## CLI Syntax
+# CLI Syntax
+Before you start reading this chapter I highly recommend you the read the [POSIX Utility Conventions][POSIX Utility Conventions]
+it is about 10-20 min read which will provide a solid base when it comes to clis and their syntax.
```

Comment:
Capitalize the "it" at the beginning, and rephrase to this:
"**It** is **approximately a** 10-20 minute read"

Capitalize the **CLI** in **CLIs**:
"when it comes to **CLIs** and their syntax."